src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/utils.ts

```diff
@@ -2,6 +2,12 @@
 // Copyright (C) 2023 Gemeente Amsterdam
 import type { StandardTextDetailData, StandardTextForm } from './types'
 
+export const isNumeric = (str?: string) => {
```

Comment:
I would really recommend to remove this, just: str && parseInt(str) is do much cleaner.

Reply:
Done